.eslintrc.json

```diff
@@ -1,36 +1,68 @@
 {
-    "env": {
-        "browser": true,
-        "es6": true
-    },
-    "plugins": [
-        "security"
```

Comment:
This should be kept, along with:
```json
"extends": [
        "plugin:security/recommended"
]
```

Reply:
They must not have been in runner, as I didn't explicitly removed them